packages/ra-ui-materialui/src/button/CreateButton.tsx

```diff
@@ -63,6 +64,14 @@ const CreateButton = (inProps: CreateButtonProps) => {
     });
     const createPath = useCreatePath();
     const translate = useTranslate();
+    const getResourceLabel = useGetResourceLabel();
+    const label =
+        labelProp ??
```

Comment:
This is a regression: if a user provides a custom label, it is no longer translated. Same for the other buttons.